frictionless/portals/github/control.py

```diff
@@ -12,17 +13,57 @@ class GithubControl(Control):
 
     # State
 
-    user: Optional[str] = None
+    apikey: Optional[str] = os.environ.get("GITHUB_ACCESS_TOKEN", None)
+    """NOTE: add docs"""
+
+    email: Optional[str] = os.environ.get("GITHUB_EMAIL", None)
+    """NOTE: add docs"""
+
+    formats: Optional[List[str]] = ["csv"]
```

Comment:
Let's have ['csv', 'xlsx', 'xls'] by default and also we might have it as DEFAULT_FORMATS const